.github/workflows/solidity.yml

```diff
@@ -1,79 +1,222 @@
-##
-# NOTE: any changes to the event triggers or the paths here should be reflected in:
-#       solidity-paths-ignore.yml
-##
-
 name: Solidity
 
 on:
   push:
-    branches:
-      - staging
-      - trying
-      - rollup
-    paths:
```

Comment:
Want to do something similar to this https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/integration-tests.yml#L26-L34 so we only run this workflow when solidity file are modified?